Mathlib/Tactic/StacksAttribute.lean

```diff
@@ -43,48 +43,83 @@ It extends the `Tag` environment extension with the data `declName, tag, comment
 def addTagEntry {m : Type → Type} [MonadEnv m] (declName : Name) (tag comment : String) : m Unit :=
   modifyEnv (tagExt.addEntry · { declName := declName, tag := tag, comment := comment })
 
-/--
-The syntax for a Stacks tag: it is an optional number followed by an optional identifier.
-This allows `044Q3` and `GH3F6` as possibilities.
--/
-declare_syntax_cat stackTag
+open Parser
+
+/-- `stacksTag` is the node kind of Stacks Project Tags: a sequence of digits and
+uppercase letters. -/
+abbrev stacksTagKind : SyntaxNodeKind := `stacksTag
+
+/-- The main parser for Stacks Project Tags: it accepts any sequence of 4 digits or
+uppercase letters. -/
```

Comment:
Oops. This should be updated.

Reply:
This is already up to date, right?

Reply:
Despite predating your change, I still think my comment holds. Since you push errors to the parser state with lower case letters, it won't make a syntax node. 

Thought perhaps this is quibbling over little. 

I think saying to the user that it accepts any alphanumeric is confusing to the user who sees an error from the parser with lower case letters

Reply:
Ugh. I need to read. Never mind